backend/backend/gmaps.py

```diff
@@ -4,10 +4,13 @@
 from typing import Callable, Iterable, TypedDict
 import logging
 
+import pytz
+from datetime import datetime, timedelta, time
+from timezonefinder import TimezoneFinder
 import tqdm.auto as tqdm
 import requests
 
-from .location import Location
+from location import Location
```

Comment:
```suggestion
from backend.location import Location
```
Let's use the full import path here as in other places to make it clear this is a local module